index.html

```diff
@@ -172,6 +172,7 @@ <h2>Contact me</h2>
 			<form action="https://formspree.io/f/xqknezyl" method="POST" id="form">
 				<label>
 					<input
+						id="name"
 						class="sm-input"
 						required
 						maxlength="30"
```

Comment:
### Required changes ♻♻

- [ ] When the user changes the content of any input field, the data is saved to the local storage.

- [ ] When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.

> I've checked your project and  is not completely fulfilling these requirements named above, you can check this example live project of this same topic, on their contact form section and see the difference between this [example](https://behnam1369.github.io/Portfolio/index.html) project with yours, and
![example](https://user-images.githubusercontent.com/54015740/151441019-21862867-697e-44b4-9955-fb427b2fdb24.png)
> as you can see in this example, their contact form allows us to see these local storage options to autofill, which we don't have in your project and would be good to implement since is part of the requirements for approval,  also when we click on the button to submit this information filling up the contact information, the button isn't functional, therefore I can't be redirected to the form spree page saying that my information has been saved and sent to the user I'm trying to get in touch with, in this case, your group.

- [ ] Please implement the changes necessary so your project can be soon approved, you're almost there 💪😁♻